library/ix-dev/community/mealie/upgrade_strategy

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python3
+import json
+import re
+import sys
+
+from catalog_update.upgrade_strategy import semantic_versioning
+
+
+RE_STABLE_VERSION = re.compile(r'v\d+\.\d+\.\d+')
+
+
+def newer_mapping(image_tags):
+    key = list(image_tags.keys())[0]
+    tags = {t.strip('v'): t for t in image_tags[key] if RE_STABLE_VERSION.fullmatch(t)}
```

Comment:
I am doubtful this will work..it will work similarly with `v1.0.0-RC1.1` and `v1.0.0-BETA`

Reply:
Plan is to only automatically upgrade to the next "stable" release.
RC versions don't have a specific format, so we can't really account for that.
I'm hoping that there won't be another RC release upstream, but if it does we can upgrade that one manual once.